internal/container/docker.go

```diff
@@ -140,8 +183,48 @@ func (e *dockerImpl) ContainerRun(ctx context.Context, image string, flags *RunF
 	return string(buf), nil
 }
 
+func parseContainers(buf []byte) ([]Container, error) {
+	result := []Container{}
+	decoder := json.NewDecoder(bytes.NewBuffer(buf))
+	for decoder.More() {
+		c := map[string]any{}
```

Comment:
I think just any should work the same
```suggestion
		c := map[string]any
```

Reply:
your suggestion resolves to a type as oppose to an instance
<img width="619" alt="Screenshot 2024-06-21 at 09 23 42" src="https://github.com/mongodb/mongodb-atlas-cli/assets/1765216/f0fabc03-0d12-4d49-b061-d07a8b90919e">